front/app/containers/Admin/projects/project/timeline/edit.tsx

```diff
@@ -320,12 +320,22 @@ const AdminProjectTimelineEdit = () => {
             <Error apiErrors={errors && errors.title_multiloc} />
           </SectionField>
           <SectionField>
-            <ParticipationContext
-              phase={!isNilOrError(phase) ? { data: phase } : undefined}
-              onSubmit={handleParticipationContextOnSubmit}
```

Comment:
For some reason this component doesn't rerender when the phase data is finally loaded, so the ParticipationContext falls back to default values. This actually prevents editing of existing phases. A quick solution I found is to just separate it into 2 components. I think it's pretty important to get this fix out, so maybe this could be a temporary solution while I figure out what's going on?